BaragonService/src/main/java/com/hubspot/baragon/service/managers/ElbManager.java

```diff
@@ -155,18 +155,18 @@ private boolean isVpcOk(BaragonAgentMetadata agent, LoadBalancerDescription elb)
   }
 
   public void syncAll() {
-    List<LoadBalancerDescription> elbs;
+    List<LoadBalancerDescription> elbs = elbClient.describeLoadBalancers().getLoadBalancerDescriptions();
```

Comment:
not familiar with the client, but there's no concerns with limits / paging here, right?